summoner-cli/src/Summoner/Template/GitHub.hs

```diff
@@ -121,49 +121,121 @@ gitHubFiles Settings{..} = concat
         , "    branches: [master]"
         , ""
         , "jobs:"
-        , "  build:"
-        , "    name: ghc ${{ matrix.ghc }}"
-        , "    runs-on: ubuntu-16.04"
+        ]
+        <> memptyIfFalse settingsCabal ghActionsCabal
+        <> memptyIfFalse settingsStack ghActionsStack
+
+    ghcActionsCheckoutVersion = "@v2.3.3"
```

Comment:
Is this place ok for keeping the versions?

Reply:
I think it's okay. As they are only used here, so no need to move to some external module.